admin/views/pupil-status/view-pupil-status.ejs

```diff
@@ -2,37 +2,274 @@
             </p>
-            <table id="pupil-status" name="filterablePupilsList" class="govuk-table govuk-spacious">
-                <caption class="govuk-body govuk-table__caption">List of pupils with statuses</caption>
-                <thead class="govuk-table__head">
-                <tr class="govuk-table__row">
-                    <th scope="col" class="govuk-table__header govuk-!-width-one-half">
-                        <a class="no-underline sortingLink">Pupil<span class="sort-icon asc"><span>Sort by pupil</span></span></a>
-                    </th>
-                    <th scope="col" class="govuk-table__header govuk-!-width-one-quarter">
-                        <a class="no-underline sortingLink">Status<span class="sort-icon"><span>Sort by status</span></span></a>
-                    </th>
-                </tr>
-                </thead>
-                <% pupils.forEach((p) => { %>
-                    <tr class="govuk-table__row">
-                        <td scope="row" class="govuk-table__cell">
-                            <span class="govuk-!-font-weight-bold name-text-wrap"><%= p.fullName %></span>
-                            <% if (p.showDoB) { %>
-                                <div class="govuk-font-greyed-out govuk-!-font-size-16">Date of birth: <%= p.dateOfBirth %></div>
-                            <% } %>
-                        </td>
-                        <td class="govuk-table__cell"><%= p.status %></td>
-                    </tr>
-                <% }) %>
-                </tbody>
-            </table>
+            <div id="red-card" class="govuk-panel govuk-panel custom-card red-card">
+                <div class="govuk-panel__body custom-card-small-text">
+                    Checks with errors
+                </div>
+                <h1 class="custom-card-large-text">
+                    <%= pupilStatusViewData.pupilsWithErrorsCount %>
+                </h1>
+                <div class="govuk-panel__body custom-card-small-text">
+                    of <%= pupilStatusViewData.totalPupilsCount %> pupils
+                </div>
+            </div>
+            <div id="orange-card" class="govuk-panel govuk-panel custom-card orange-card">
+                <div class="govuk-panel__body custom-card-small-text">
+                    Pupils that have not started yet
+                </div>
+                <h1 class="custom-card-large-text">
+                    <%= pupilStatusViewData.pupilsNotStartedCount %>
+                </h1>
+                <div class="govuk-panel__body custom-card-small-text">
+                    of <%= pupilStatusViewData.totalPupilsCount %> pupils
+                </div>
+            </div>
+            <div id="blue-card" class="govuk-panel govuk-panel custom-card blue-card">
+                <div class="govuk-panel__body custom-card-small-text">
+                    Pupils not taking the check
+                </div>
+                <h1 class="custom-card-large-text">
```

Comment:
Using multiple `H1` will make it very hard for screen reader users.  Ideally, there should be a single `H1` tag.